propulsion-summary-projector/Program.fs

```diff
@@ -155,7 +155,7 @@ module CmdParser =
         member __.Heartbeat =               a.GetResult(HeartbeatTimeout,1.5) |> TimeSpan.FromSeconds
         member __.Connect(log: ILogger, storeLog: ILogger, appName, connectionStrategy) =
             let s (x : TimeSpan) = x.TotalSeconds
-            log.Information("EventStore {host} heartbeat: {heartbeat}s Timeout: {timeout}s Retries {retries}", __.Host, s __.Heartbeat, s __.Timeout, __.Retries)
+            log.Information("EventStore {host}:{port} heartbeat: {heartbeat}s Timeout: {timeout}s Retries {retries}", __.Host, __.Port, s __.Heartbeat, s __.Timeout, __.Retries)
```

Comment:
Does this look clear for both the Some and None cases? It may be worth doing
`(match __.Host,__.Port with h, Some p -> sprintf "%s:%d" h p | h,None ->h)` instead?
(This would also work better if we make alterations to Equinox.EventStore.Connector to permit the host value to have an embedded `:port`)

Reply:
👍 about helper function for pretty print it